src/lib/extractor-shared/extractor_shared/csv_generation/write.py

```diff
@@ -17,6 +17,7 @@
 SUBMISSION_ROOT_DIRECTORY = "section={id1}/assignment={id2}/submissions/"
 USER_ACTIVITY_ROOT_DIRECTORY = "section={id}/user-activities/"
```

Comment:
I think `SYSTEM_ACTIVITY_ROOT_DIRECTORY` should replace `USER_ACTIVITY_ROOT_DIRECTORY `, so there should be only system activities and section activities, @stephenfuqua  am I right?

Reply:
Good point - "user activities" was replaced by "section activities"